app/components/GlobalError/GlobalError.js

```diff
@@ -2,6 +2,7 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import MdClose from 'react-icons/lib/md/close'
 import styles from './GlobalError.scss'
+import errorToUserFriendly from '../../lib/utils/userFriendlyErrors'
```

Comment:
change to `import errorToUserFriendly from 'lib/utils/userFriendlyErrors'`

Reply:
I'm not sure that'll work but I'll try it